tests/integration/database-services.test.ts

```
@@ -0,0 +1,831 @@
+          
+          await client.query('COMMIT');
+          resolve({ success: true, customerId });
+        } catch (error) {
+          await client.query('ROLLBACK');
+          throw error;
+        } finally {
+          client.release();
+        }
+      });
+
+      expect(mockClient.query).toHaveBeenCalledWith('BEGIN');
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)